src/sentry/grouping/strategies/base.py

```diff
@@ -354,6 +357,11 @@ class NewStrategyConfiguration(StrategyConfiguration):
     NewStrategyConfiguration.delegates = dict(base.delegates) if base else {}
     NewStrategyConfiguration.initial_context = dict(base.initial_context) if base else {}
     NewStrategyConfiguration.enhancements_base = base.enhancements_base if base else None
+    if base and base.fingerprinting_bases is not None:
```

Comment:
This case is not covered afaict
<img width="121" alt="image" src="https://github.com/getsentry/sentry/assets/44410/67f011dc-271f-4234-ac88-ffe2d27d7a34">